elasticdl/python/common/tensor_utils.py

```diff
@@ -9,18 +9,21 @@ def merge_indexed_slices(*args):
 
 
 def deduplicate_indexed_slices(values, indices):
-    """Sums `values` associated with any non-unique `indices`.
+    """
+    Sum up the values associated with duplicated indices in the IndexedSlices.
```

Comment:
No `IndexedSlices` in this method.
Can rephrase as:
Sums up values associated with the same indices and returns unique indices with corresponding summed values.

Reply:
Good suggestion, updated.